examples/models/llama/export_llama_lib.py

```diff
@@ -951,7 +954,9 @@ def _load_llama_model(
             use_kv_cache,
             use_sdpa_with_kv_cache,
             enable_dynamic_shape,
-            model.params,
+            model.max_seq_len,
```

Comment:
Previously all the models were getting it from `model.params`. Is it guaranteed that all models will have these available on them as attributes directly? Hopefully CI catches if they don't.

Reply:
Yeah, hopefully CI catches but should be okay since there's only two `model.py`s using this `export_llama_lib` atm and both have it defined